### DISCUSSION TITLE
Webhook for ticket notification

### DISCUSSION THREAD

**Discussion by floatingpurr** [django-helpdesk/django-helpdesk#1106] [Q&A] [answered]
Hi, I'm wondering if there's a way to push ticket notifications (e.g., creation, update, etc...) to an external service with a webhook or a similar mechanism.

**Comment by uhurusurfa** [comment, marked as answer]
Currently there is no support for webhooks.
If you have the knowledge and the time, feel free to raise a PR to start support.
